.github/workflows/ci.yml

```diff
@@ -89,3 +89,53 @@ jobs:
         - name: Run pipeline with profile ${{ matrix.tests }}
           run: |
             nextflow run ${GITHUB_WORKSPACE} -profile ${{ matrix.tests }},docker
+
+  nonfree:
+      name: Run non-free tests
```

Comment:
I would suggest to use a different name here, maybe just "Run netMHC tool tests"

Reply:
Done.